lib/src/resource/manager.dart

```diff
@@ -21,7 +21,18 @@ class ResourceManager {
 
   /// Get a resource with the given [ResourceName]
   T getResource<T>(ResourceName name) {
-    final resource = resources[name];
+    Resource? resource;
+    if (resources.containsKey(name)) {
+      resource = resources[name];
+    } else {
+      final resourcesWithoutRemotes = resources.map((rn, res) {
```

Comment:
(q) Potentially out of scope for a hotfix but, should `resourcesWithoutRemotes` live on the `ResourceManager` such that we don't have to construct this new map potentially every time we call `getResource`?

Reply:
Did it anyway -- it's a good thought